### DISCUSSION TITLE
Uncaught (in promise) TypeError: Cannot read properties of undefined

### DISCUSSION THREAD

**Discussion by karangoraniya** [vercel/community#2303] [Help]
I have installed a fresh Next.js template without installing any additional packages, and I'm encountering an error. Same error facing on deployed site. 

```quests.90898c83.js:44 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'length') at requests.90898c83.js```

**Comment by amyegan** [comment]
Hi @karangorania! Please share a link to the template you used 🙏

**Reply by karangoraniya** [reply]
@amyegan 
means this create new app template default one`npx create-next-app@latest`


this is my deployed site 

[UDtools](https://www.udtools.app/)
[Bridge Pass](https://bridgepass.vercel.app/)

first i think any package error is this but it showing in with fresh app also

**Reply by karangoraniya** [reply]
@amyegan 

I tried again and found out that an uncaught error is only showing in the Brave browser.

**Reply by karangoraniya** [reply]
@amyegan error solved auto after 2 days
